azure-pipelines-full-tests.yml

```diff
@@ -92,3 +72,99 @@ jobs:
         pythonVersion: '$(python.version)'
         profile: '2018-03-01-hybrid'
         fullTest: true
+
+- job: AutomationFullTestPython36ProfileLatest
+  displayName: Automation Full Test Python36 Profile Latest
+  timeoutInMinutes: 9999
+  strategy:
+    maxParallel: 8
+    matrix:
+      8_1:
+        Instance: 8_1
```

Comment:
Extract `8` as a standalone parameter (instance count) for better extensibility (no need to update all strings if we extend to 10 instances.

Reply:
Thanks for the suggestion.
Split parameter `instance` into `instance_cnt` and `instance_idx`.

Reply:
What does `x` in `idx` stands for?